prov/tcp/src/tcpx_ep.c

```diff
@@ -731,6 +803,12 @@ int tcpx_passive_ep(struct fid_fabric *fabric, struct fi_info *info,
 	_pep->cm_ctx.cm_data_sz = 0;
 	_pep->sock = INVALID_SOCKET;
 
+	ret = tcpx_set_port_range(_pep);
+	if (ret == -FI_EINVAL) {
+		FI_WARN(&tcpx_prov, FI_LOG_EP_CTRL,"Invalid info\n");
+		return -FI_EINVAL;
+	}
```

Comment:
Read the port range values where other provider environment variables are read, likely in init somewhere.